tests/runner/Polynetwork/message_1.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
I'm happy with the number of tests, but I have no idea how to determine whether the expected output is correct. Are these tests (and their expected output) copied from the Solidity repo, or did we write them ourselves?

Reply:
All tests from the solidity repo were ported (except for ones that dealt with upgrading the contract). In addition, I have a bunch of unit tests of my own as well.

Reply:
That's fine, then.